### ISSUE TITLE
Languages

### ISSUE THREAD

**Issue by thebigsmileXD** [thebigsmileXD/BedWars#38]
I planned to implement languages into the plugin, fistly, to nuke the German-Edition fork, secondly to allow anyone to use languages

**Comment by SuperYYT** [comment]
404 not found

**Comment by thebigsmileXD** [comment]
@SuperYYT Yes, there currently is no translation branch, and no translation file. I am going to add it with version 2.0.0

**Comment by SuperYYT** [comment]
> @SuperYYT Yes, there currently is no translation branch, and no translation file. I am going to add it with version 2.0.0

ok，thank you
